src/main/resources/persistence.properties

```diff
@@ -4,7 +4,8 @@ hibernate.archive.autodetection = class
 
 javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
 javax.persistence.jdbc.user = moodcat
-hibernate.hbm2ddl.auto = create-drop
+#hibernate.hbm2ddl.auto = create-drop
+hibernate.hbm2ddl.auto = update
```

Comment:
This change already is in master, but without the comment. Maybe rebase on master to be able to auto-merge?